doc/attr.xml

```diff
@@ -659,7 +659,26 @@ rec( comps := [ [ 3 ], [ 1, 2 ] ], id := [ 2, 2, 1 ] )
 </ManSection>
 <#/GAPDoc>
 
-#
+<#GAPDoc Label="DigraphNrStronglyConnectedComponents">
+<ManSection>
+  <Attr Name="DigraphNrStronglyConnectedComponents" Arg="digraph"/>
+  <Returns>A non-negative integer.</Returns>
+  <Description>
+    This function returns the number of strongly connected components of
+    <A>digraph</A>.  Note that this is no more efficient than calling <Ref
+    Attr="DigraphStronglyConnectedComponents"/>.
```

Comment:
If you intend for this to be the end of a paragraph, you need a `<P/>` tag at the end here.